api/app/src/routes/medical/document.ts

```diff
@@ -63,32 +67,25 @@ router.post(
 /** ---------------------------------------------------------------------------
  * GET /document
```

Comment:
Just noticed this is wrong. Once we decide on the name, could you fix it, please?